net.sourceforge.m64py.M64Py.yml

```diff
@@ -0,0 +1,140 @@
+app-id: net.sourceforge.m64py.M64Py
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.7'
+
+command: m64py
+
+finish-args:
+  - --share=ipc
+  - --device=all
+  - --socket=pulseaudio
+  - --socket=fallback-x11
+  - --socket=wayland
+
+build-options:
+  env:
+    - BASEAPP_REMOVE_WEBENGINE=1
+
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+  - rm -rf /app/lib/python/site-packages/PyQt6/QtBluetooth*
```

Comment:
why are you deleting these?

Reply:
Well, why waste the space, I only use QtCore, QtGui, and QtWidgets.

Reply:
it's a bit slippery slope, if one of those starts depending on the ones you delete.

Reply:
For now, only QtDBus is needed, and if anything ever changes it cannot go unnoticed. Several megabytes, but it is worth the trouble I think.